.github/workflows/clean_up_package_registry.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+name: Clean up packages in registry
+
+on:
+  workflow_dispatch:
+  # Runs always Sunday Midnight
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  clean_all:
```

Comment:
Suggestion: clean_all -> clean_up_package_registry

Reply:
This is the job, not touching it